sdk/dotnet/Dagger.SDK.Tests/GraphQL/QueryBuilderTest.cs

```diff
@@ -0,0 +1,83 @@
+using Dagger.SDK.GraphQL;
+
+namespace Dagger.SDK.Tests.GraphQL;
+
+public class QueryBuilderTest
+{
+    [Fact]
+    public void TestSelect()
+    {
+        var query = QueryBuilder
+            .Builder()
+            .Select("container")
+            .Build();
+
+        Assert.Equal("query{container}", query);
+    }
+
+    [Fact]
+    public void TestSelect_WithArgument()
+    {
+        var query = QueryBuilder
+                        .Builder()
+                        .Select("container")
```

Comment:
This will end up something like the Go SDK? 

`var container = dagger.Client.Container()` <- QueryBuilder

`container.From("image").WithEnv("...").WithExec("...")`

Reply:
Yes, the SDK API will construct the graphql query throuh this builder.